emloop/cli/common.py

```diff
@@ -244,12 +262,12 @@ def run(config: dict, output_root: str, restore_from: str=None, eval: Optional[s
     :param config: configuration
     :param output_root: dir under which output_dir shall be created
     :param restore_from: from whence the model should be restored (backend-specific information)
-    :param eval: optional name of the stream to be evaluated
+    :param eval_stream: Optional name of the stream to be evaluated. If set, extra_streams of main_loop are ignored.
     """
 
     output_dir = dataset = model = hooks = main_loop = None
 
-    config_copy = deepcopy(config)
+    config_copy = copy.deepcopy(config)
```

Comment:
I wonder why did you change this (from `deepcopy` to `copy.deepcopy`)

Reply:
I actually like this one. :wink: 
```
Namespaces are one honking great idea -- let's do more of those!
```